src/constants.py

```diff
@@ -0,0 +1,7 @@
+from sklearn.metrics import get_scorer
+
+from evaluate.scorers import custom_scorers
+
+
+SKLEARN_METRICS = {metric: get_scorer(metric) for metric in ["roc_auc"]}
+METRICS = dict(custom_scorers, **SKLEARN_METRICS)
```

Comment:
This could introduce bugs when sklearn_metrics and cutsom_scorers have same key. In this case the sklearn scorer will overwrite the custom_scorers scorer. Which is behaviour you do want. Maybe we can built in a check, warning or something to catch this